### PR TITLE
chore: fix some typos in comment

### PR DESCRIPTION
**Pull request by InventiveCoder**

### CODE REVIEW

File: scripts/ft-benchmark-data-sender.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def get_commit() -> tuple[str, datetime]:
 
 def commit_to_db(data: dict) -> None:
     print(data)
-    chdir(os.path.expanduser("~/nearcore/benchmarks/continous/db/tool"))
+    chdir(os.path.expanduser("~/nearcore/benchmarks/continuous/db/tool"))
```
Comment (wacban) [inline scripts/ft-benchmark-data-sender.py:89]: cc @mooori @nagisa FYI in case any other tooling depends on this typo.

**Review by wacban — APPROVED** [review]

**Comment by wacban** [comment]
@InventiveCoder Thank you for the contribution!

File: scripts/ft-benchmark-data-sender.py (hunk 1 above)
Comment (mooori) [reply scripts/ft-benchmark-data-sender.py:89]: The directory `benchmarks/continous` should be renamed accordingly. I can send a follow up PR for that.

Comment (wacban) [reply scripts/ft-benchmark-data-sender.py:89]: @InventiveCoder This PR merge failed anyway for unrelated reasons, can you either do the suggestion above or revert the relevant changes?

Comment (mooori) [reply scripts/ft-benchmark-data-sender.py:89]: To do the change here it is sufficient to:

```
git mv benchmarks/continous benchmarks/continuous
```

Comment (InventiveCoder) [reply scripts/ft-benchmark-data-sender.py:89]: I see there is already a PR to modify the directory name. Do I need to roll back the changes here?

Comment (InventiveCoder) [reply scripts/ft-benchmark-data-sender.py]: Reverted to be able to merge without conflicts

**Comment by InventiveCoder** [comment]
Anything need I to do?

**Pull request merged**